devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -924,6 +924,25 @@ bool WholeBodyDynamicsDevice::loadSettingsFromConfig(os::Searchable& config)
     if (settings.useJointVelocity) { yInfo() << "wholeBodyDynamics: jointVelFilterCutoffInHz: " <<   settings.jointVelFilterCutoffInHz << " Hz."; }
     if (settings.useJointAcceleration) { yInfo() << "wholeBodyDynamics: jointAccFilterCutoffInHz: " <<   settings.jointAccFilterCutoffInHz << " Hz."; }
     
+    if ( !(prop.check("skipResetOffset") && prop.find("skipResetOffset").isBool()) )
+    {
+        yWarning() << "wholeBodyDynamics: skipResetOffset bool parameter missing.";
+        yWarning() << "wholeBodyDynamics: setting skipResetOffset to the default value of false, please remember to resetOffset through the rpc port.";
```

Comment:
This is misleading. On the real robot, you don't have to remember to call `resetOffset` .

Reply:
Noted. I will remove that line.

Reply:
Addressed in https://github.com/robotology/whole-body-estimators/pull/72/commits/b495053958d918d01711010f7e32f1ac651fc29c